src/plone/restapi/services/relations/get.py

```diff
@@ -0,0 +1,148 @@
+from AccessControl.SecurityManagement import getSecurityManager
+from collections import defaultdict
+from plone.restapi.interfaces import ISerializeToJsonSummary
+from plone.restapi.serializer.converters import json_compatible
+from plone.restapi.services import Service
+from Products.CMFCore.utils import getToolByName
+from Products.CMFPlone.relationhelper import get_relations_stats
```

Comment:
If this targets also Plone 5.2 we need to rework this import and write the method directly here because this does not exist in Plone 5.2.x